src/Documentation/Advanced-Concepts/Heterogeneous-Silos.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Heterogeneous silos
+---
+
+# Heterogeneous silos
+On a given cluster, silos can support a different set of grain types:
```

Comment:
In the first commit there is the png in the src/ dir, and in the second one it is the 'html'version (check the file list, it will not appear in GitHub diff since).

Anyway I added the visio file also